src/apps/Map/components/WebglMap/WebglMap.tsx

```diff
@@ -204,47 +218,30 @@ class Map extends PureComponent<Props, State> {
   };
 
   registerMouseHoverHandler = () => {
-    const hbiClickableLayers = [
+    const clickableLayers = [
       config.apps.map.layers.hbi.overlayLine,
       config.apps.map.layers.hbi.xOverlay,
+      config.apps.map.layers.projects.overlayLine,
     ];
-    const projectsTarget = config.apps.map.layers.projects.overlayLine;
-
-    const setCursorPointer = () => {
-      this.map.getCanvas().style.cursor = 'pointer';
-    };
-    const setCursorNone = () => {
-      this.map.getCanvas().style.cursor = '';
-    };
 
     // Events 'mouseenter' & 'mouseleave' didn't worked
     // when hovering from clickable layer to another clickable layer
     this.map.on('mousemove', (e) => {
       const features = this.map.queryRenderedFeatures(e.point);
```

Comment:
We should maybe integrate a layer-filter , guess this is more performant, see https://maplibre.org/maplibre-gl-js-docs/api/map/#map#queryrenderedfeatures -> `options.filter`

Reply:
@ohrie I merged the change as part of your PR. Performance looked OK though, so maybe something for later.